helix-query.yaml

```diff
@@ -7,6 +7,7 @@ indices:
     exclude:
       - /drafts/**
       - /block-library/**
+      - /_template-*
```

Comment:
I think the underscore is ignored, maybe it has to be omitted.

Reply:
I'll test it.